lib/src/cmaf/ham/manifestParser.ts

```diff
@@ -1,14 +1,29 @@
-async function readHLS(manifestUrl: string): Promise<string> {
-    const response = await fetch(manifestUrl, {
-        headers: {
-            'Content-Type': 'application/vnd.apple.mpegurl',
-        }
-    });
-    return response.text();
+import { parseMpd } from '../utils/dash/mpd';
+import { DashManifest } from '../utils/dash/DashManifest';
+import { mapMpdToHam } from './hamMapper';
+
+export async function readHLS(manifestUrl: string): Promise<string> {
+	const response = await fetch(manifestUrl, {
+		headers: {
+			'Content-Type': 'application/vnd.apple.mpegurl',
+		},
+	});
+	return response.text();
 }
 
 
 export async function m3u8toHam(url: string) {
-    
+
 }
 
+
+export async function mpdToHam(manifest: string) {
+	let dashManifest: DashManifest | undefined;

```

Comment:
We should have a function for reading dash manifests and then converting into ham object

Reply:
Let's create a new ticket for the new changes needed here. @DuckyCB